scripts/ansible/create-policy.yaml

```diff
@@ -0,0 +1,17 @@
+---
+- name: Create policy on AWS using localhost
+  hosts: localhost
+  connection: local
+  tasks:
+# Create Policy ex nihilo
```

Comment:
Didn't get what this comment means

Does this require anything already set on the machine running the playbook? If yes then I think having that as a comment would help people (like me) which is not familiar yet with iam_managed_policy Ansible module.